wallet-new/src/Cardano/Wallet/Kernel/BListener.hs

```diff
@@ -170,20 +170,34 @@ applyBlock pw@PassiveWallet{..} b = do
                                  <> "restoring " <> show rootId)
               restoreKnownWallet pw rootId
 
-          -- Beginning with the oldest missing block, update each lagging account.
-          let applyOne (block, toAccts) = runExceptT (applyOneBlock k (Just toAccts) block)
           case toApply of
               []       -> return () -- nothing to do!
-              (bk:bks) -> do
-                  failures <- mapM applyOne (getOldestFirst $ gatherAcctsPerBlock (bk :| bks))
+              (bk:bks) -> backfilling k $ gatherAcctsPerBlock (bk :| bks)
 
-                  case NEM.fromMap . Map.unions . map NEM.toMap . lefts $ failures of
-                      Nothing       -> return ()                         -- OK, no failures, we are done!
-                      Just moreErrs -> handleApplyBlockErrors k moreErrs -- Try again, better luck next time.
+      -- Beginning with the oldest missing block, update each lagging account.
+      backfilling :: Node.SecurityParameter
+                  -> OldestFirst [] (ResolvedBlock, Set HdAccountId)
+                  -> IO ()
+      backfilling k acctsPerBlock = do
+          _walletLogMessage Info $ "Wallet is behind node by "
```

Comment:
I am tempted to put this as a `Warning`, so that filtering the logs with severity `Warning+` would allow us to see only the interesting bits without all the surrounding noise. That's a minor one, though!